### PR TITLE
Updated academic section of developer guide.

### PR DESCRIPTION
**Pull request by luziyi9898**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -52,6 +52,47 @@ in Bookmark mode.
 
 
 
+**Academic Component**
```
Comment (lingsihui) [inline docs/DeveloperGuide.md]: can change to ##Academic Component!

**Review by lingsihui — APPROVED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (farice9) [reply docs/DeveloperGuide.md]: I'll merge and help edit this since I'm working on it, thanks for pointing!

**Review by farice9 — APPROVED** [review]
LGTM let's get it!

**Comment by farice9** [comment]
Merging this despite failing check. The check failed due to a JUnit test from @slightlyharp .

**Pull request merged**